File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/CommentService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbComment;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface CommentService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbComment> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbComment comment);


	/**
	 * 修改
	 */
	public void update(TbComment comment);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbComment findOne(Long comId);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] comIds);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbComment comment, int pageNum, int pageSize);
	
}

File: client_manage_web/src/main/webapp/js/service/commentService.js
//服务层
app.service('commentService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../comment/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../comment/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../comment/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../comment/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../comment/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../comment/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../comment/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});